.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -44,7 +44,7 @@ Please do not remove items from the checklist
       Link: <!-- example https://github.com/kubernetes-sigs/kueue/releases/tag/v0.1.0 -->
 - [ ] For major or minor releases, merge the `main` branch into the `website` branch to publish the updated documentation.
 - [ ] Send an announcement email to `sig-scheduling@kubernetes.io` and `wg-batch@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`. Link: <!-- example https://groups.google.com/a/kubernetes.io/g/wg-batch/c/-gZOrSnwDV4 -->
-- [ ] Update `README.md`, `CHANGELOG`, `site/config.toml`, `charts/kueue/Chart.yaml` (`appVersion`) and `charts/kueue/values.yaml` (`controllerManager.manager.image.tag`) in `main` branch: <!-- example #774 -->
+- [ ] Update `README.md`, `CHANGELOG`, `site/config.toml`, `charts/kueue/Chart.yaml` (`appVersion`) and `charts/kueue/values.yaml` (`controllerManager.manager.image.tag`,`SECURITY-INSIGHTS.yaml`) in `main` branch: <!-- example #774 -->
```

Comment:
```suggestion
- [ ] Update `README.md`, `CHANGELOG`, `site/config.toml`, `charts/kueue/Chart.yaml` (`appVersion`) and `charts/kueue/values.yaml` (`controllerManager.manager.image.tag`), `SECURITY-INSIGHTS.yaml` (`last-updated`, `last-reviewed`, `commit-hash`, `project-release`) in `main` branch: <!-- example #774 -->
```

Did I miss anything?

Or maybe you can put each file into a separate item, for better visibility.

Reply:
In the latest commit, I've separated the files.